ops/production-deploy.tmpl.yaml

```diff
@@ -5,11 +5,11 @@ imagePullSecrets:
 
 resources:
   limits:
-    memory: "4Gi"
```

Comment:
this should be 2/4GB like other web processes. I can understand matching UTK, but we're not importing massively here so I think the worker can be much smaller as well